buildozer/targets/android.py

```diff
@@ -245,11 +245,14 @@ def apache_ant_dir(self):
     @property
     def sdkmanager_path(self):
         sdkmanager_path = join(
-            self.android_sdk_dir, 'tools', 'bin', 'sdkmanager')
+            self.android_sdk_dir, 'tools', 'bin',
+            'sdkmanager.bat'
+            if platform in ('win32', 'cygwin')
+            else 'sdkmanager')
```

Comment:
Just for readability, how about defining the executable (`sdkmanager` or `sdkmanager.bat`) outside of the `join()`?

Reply:
Good idea. Done.